test/OriginationController.ts

```diff
@@ -227,7 +227,7 @@ describe("OriginationController", () => {
             const MockOriginationController = await hre.ethers.getContractFactory("MockOriginationController", other);
             await expect(
                 hre.upgrades.upgradeProxy(originationController.address, MockOriginationController),
-            ).to.be.revertedWith("Ownable: caller is not the owner");
+            ).to.be.revertedWith("AccessControl: account 0x88cde74d0d35369cf2253ef8353581f750b49f1f is missing role 0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775");
```

Comment:
Remember, we shouldn't be hardcoding accounts or roles here....I have these fixed in my PR so I'd just revert `OriginationController.ts` back to its original state